app-modules/workflow/src/Jobs/CareTeamWorkflowActionJob.php

```diff
@@ -95,6 +99,13 @@ public function handle(): void
                     $workflowRunStepRelated->save();
                 });
 
+            if (WorkflowSequentialExecutionFeature::active()) {
+                $this->markStepCompletedAndScheduleNext($this->workflowRunStep);
```

Comment:
Every time this method is called, you pass in `$this->workflowRunStep`. Maybe the param can be removed, and you just access `$this->workflowRunStep` from inside the trait method? Then there is less code to write for each action job.